prelude/__internal__/HitPoint.mad

```diff
@@ -0,0 +1,21 @@
+export data HitPoint = FunctionHit String String
+                     | LineHit String Number
+
+#-
+const __HIT_POINTS = [];
+global.__hp = (astPath, type, line, id) => {
+  if (type === "function") {
+    __HIT_POINTS.push(FunctionHit(astPath, id))
+    __HIT_POINTS.push(LineHit(astPath, line))
+  }
+  else if (type === "line") {
+    __HIT_POINTS.push(LineHit(astPath, line))
+  }
+}
+-#
+
+
+
+hitPoints :: a -> List HitPoint
+export getHitPoints = (a) => (#- [...__HIT_POINTS] -#)
+
```

Comment:
As mentioned on Slack, I think some of the prelude stuff needs more organization, but I think this is fine for a first cut.

Reply:
Yes, we definitely agree. We need to find a longer term solution. Maybe we do a private folder in prelude or something ? And the compiler would just have a rule to exclude these from resolution when importing something ? Or maybe it's fine to just not document anything that is in Private or Internal. I don't think people would go and try to import something like `import HP from "__Internal__/HitPoint"`.